### PR TITLE
Volume for Asset and AssetOccupancy

### LINKED ISSUE #5: Scenario for Asset that only cares about total booking amount
Hi , Our team is dicussing a scenario described below, hope that you can help us figure out the best practice.

For example, there's a pool asset, which can Contain 500 people at most.
In the morning, we allow only 200 people to book the pool (08:00 ~12:00).
In the afternoon, we allow all 500 people to book the pool (14:00 ~18:00).

If we use current model , we will have to build 500 asset for this pool,
and create periods for morning (08:00 ~12:00) and afternoon(14:00 ~18:00).

Maybe there's better ways to handle this type of asset, which only cares about total booking amount?

### PR DESCRIPTION
**Pull request by gdlcf88**
Resolve #5

### CODE REVIEW

**Comment by gdlcf88** [comment]
cc @ericeric284

File: src/EasyAbp.BookingService.Domain/EasyAbp/BookingService/Assets/Asset.cs [hunk 1]
```diff
@@ -30,6 +30,11 @@ public class Asset : FullAuditedAggregateRoot<Guid>, IMultiTenant
     /// </summary>
     public virtual PeriodUsable? DefaultPeriodUsable { get; protected set; }
 
+    /// <summary>
+    /// Assets can occupy if the rest volume is enough.
+    /// </summary>
+    public virtual int Volume { get; protected set; }
```
Comment (yinchang0626) [inline src/EasyAbp.BookingService.Domain/EasyAbp/BookingService/Assets/Asset.cs:36]: @gdlcf88 
Maybe `Volume` property is belong to `AseetCategory` is better?
Or let `AseetCategory` simpler,it jsut a Category  of  `Asset` ,It doesn't matter with `Booking`?
In other words  `Asset` is not only `房間` but also `房型`?

Comment (gdlcf88) [reply src/EasyAbp.BookingService.Domain/EasyAbp/BookingService/Assets/Asset.cs:36]: @yinchang0626 - That is simpler. But it also means `房间` cannot be specified to occupy anymore if we make `房型` an `Asset`.

Comment (yinchang0626) [reply src/EasyAbp.BookingService.Domain/EasyAbp/BookingService/Assets/Asset.cs:36]: OK, I wiil treat `Aseet` as a `房型` to testing bookservice

**Pull request merged**